src/composables/baseInstrumentConfig.js

```diff
@@ -46,19 +46,26 @@ export default function baseInstrumentConfig(instrumentConfig, availableInstrume
   const rotatorModeOptions = computed(() => {
     let options = [];
     let requiredModeFields = [];
+    let defaultValues = {};
     if (hasModes('rotator')) {
       let modes = _.get(availableInstruments.value, [selectedInstrument.value, 'modes', 'rotator', 'modes'], []);
       for (let i in modes) {
         requiredModeFields = [];
+        defaultValues = {};
```

Comment:
Do you need to set this to an empty object since you did it above?

Reply:
yeah it can vary for each of the modes we are iterating through